src/components/Local.vue

```diff
@@ -149,4 +157,16 @@ export default {
   display: flex;
   justify-content: center;
 }
+.small-info {
+  position: absolute;
+  right: 0.36em;
+  bottom: 0.18em;
+  font-size: 0.85em;
+}
+/* better look for the small devices */
+@media (max-width: 720px) {
+  .small-info {
+    font-size: 0.65em;
+  }
+}
```

Comment:
ah you really are a CSS king man !!

Reply:
Nope, it's kinda basic CSS :smiley:

Reply:
I think I should stop using frameworks excessively and use my own CSS 😖